brian2wasm/__main__.py

```diff
@@ -0,0 +1,74 @@
+import argparse
+import sys
+import os
+
+def main():
+
+    parser = argparse.ArgumentParser(
+        description="Brian2WASM CLI"
+    )
+    parser.add_argument(
+        "script",
+        help="Path to the Python script to run"
+    )
+    args = parser.parse_args()
+
+    script_path = args.script
+
+    # Check if the script exists and is a Python file
+    if not os.path.isfile(script_path):
+        print(f"Error: File '{script_path}' does not exist.", file=sys.stderr)
+        sys.exit(1)
+    if not script_path.endswith(".py"):
+        print(f"Error: File '{script_path}' is not a Python script (.py).", file=sys.stderr)
+        sys.exit(1)
+
+    # Read the original script
+    with open(script_path, 'r') as f:
+        script_content = f.read()
+
+    # Get the script's directory and name
+    script_dir = os.path.dirname(script_path) or '.'
+    script_name = os.path.splitext(os.path.basename(script_path))[0]
+
+    # Check if an HTML file with the same name exists
+    html_file = f"{script_name}.html"
+    html_file_path = os.path.join(script_dir, html_file)
+    has_html_file = os.path.isfile(html_file_path)
+
+    # Inject the required lines at the top
+    if has_html_file:
+        print(f"html file found: '{html_file_path}'")
+        injection = (
+            "from brian2 import *\n"
```

Comment:
I would replace this by `from brian2 import set_device` to avoid that there are name clashes later for users that did not use `from brian2 import *`.